ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-upi-e2e.yaml

```diff
@@ -0,0 +1,1033 @@
+          echo "$INGRESS_FIP console-openshift-console.apps.$CLUSTER_NAME.$BASE_DOMAIN" >> /${TMP_ARTIFACTS}/hosts
+          echo "$INGRESS_FIP integrated-oauth-server-openshift-authentication.apps.$CLUSTER_NAME.$BASE_DOMAIN" >> /${TMP_ARTIFACTS}/hosts
+          echo "$INGRESS_FIP oauth-openshift.apps.$CLUSTER_NAME.$BASE_DOMAIN" >> /${TMP_ARTIFACTS}/hosts
+          echo "$INGRESS_FIP prometheus-k8s-openshift-monitoring.apps.$CLUSTER_NAME.$BASE_DOMAIN" >> /${TMP_ARTIFACTS}/hosts
+          echo "$INGRESS_FIP grafana-openshift-monitoring.apps.apps.$CLUSTER_NAME.$BASE_DOMAIN" >> /${TMP_ARTIFACTS}/hosts
+          echo "DNS record for api.$CLUSTER_NAME.$BASE_DOMAIN. -> $LB_FIP was NOT created!"
+          echo "DNS record for *.apps.$CLUSTER_NAME.$BASE_DOMAIN. -> $INGRESS_FIP was NOT created!"
+        fi
+
+        echo "Creating install_config.yaml"
+        # Create install-config.yaml.bkk to use as install-config.yaml because
+        # some of the next steps will delete the install-config.yaml thus we create a permanent one.
+        cat > ${ASSETS_DIR}/install-config.yaml.bkk << EOF
+        apiVersion: v1
+        baseDomain: ${BASE_DOMAIN}
+        compute:
+        - hyperthreading: Enabled
+          name: worker
+          platform: {}
+          replicas: 0
+        controlPlane:
+          hyperthreading: Enabled
+          name: master
+          platform: {}
+          replicas: 3
+        metadata:
+          name: ${CLUSTER_NAME}
+        networking:
+          clusterNetwork:
+          - cidr: 10.128.0.0/14
+            hostPrefix: 23
+          machineCIDR: ${OS_SUBNET_RANGE}
+          networkType: ${NETWORK_TYPE}
+          serviceNetwork:
+          - 172.30.0.0/16
+        platform:
+          openstack:
+            cloud: ${OS_CLOUD}
+            computeFlavor: ${OPENSTACK_FLAVOR}
+            externalNetwork: ${OPENSTACK_EXTERNAL_NETWORK}
+            lbFloatingIP: ${LB_FIP}
+
+        pullSecret: >
+          ${PULL_SECRET}
+        sshKey: |
+          ${SSH_PUB_KEY}
+        EOF
+
+
+        # Create manifest files
+        echo "Creating manifest files"
+        cp ${ASSETS_DIR}/install-config.yaml.bkk ${ASSETS_DIR}/install-config.yaml
+        TF_LOG=debug openshift-install --dir=${ASSETS_DIR} create manifests  --log-level=debug 2>&1 | grep --line-buffered -v password
+
+        #Remove Machines from manifests
+        rm -f ${ASSETS_DIR}/openshift/99_openshift-cluster-api_master-machines-*.yaml
+
+        #Remove MachineSets
+        rm -f ${ASSETS_DIR}/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+
+        #Make control-plane nodes unscheduable
+        sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" ${ASSETS_DIR}/manifests/cluster-scheduler-02-config.yml
+
+        # Create ignition-configs
+        echo "Creating ignition-configs"
+        cp ${ASSETS_DIR}/install-config.yaml.bkk ${ASSETS_DIR}/install-config.yaml
```

Comment:
this needs to come out. readding the install-config.yaml file back will cause problems because the command will reprocess it.